### PR TITLE
Matter: add items to prerequisites

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/matter.markdown [hunk 1]
```diff
@@ -104,10 +104,17 @@ Each Matter network is called a fabric. Each home automation controller that con
...
 - On the device packaging, check for both the Matter logo and for either the Wi-Fi or the Thread logo.
 - Check if the QR code is only on the packaging or if it is also on the device.
-  - If it is only on the packaging, snap a picture of the QR code and the device and store it in a save place.
+  - If it is only on the packaging, snap a picture of the QR code and the device and store the image and the numerical code in a save place.
   - If you lose the QR code and disconnect the device at some point, you won't be able to connect to that device again without the QR code.
+- Check your network settings:
+  - Make sure IPv6 is enabled in your home network. You do not need IPv6 from your <abbr title="internet service provider">ISP</abbr>, just in your local network.
+  - Make sure IPv6 is enabled in Home Assistant:
+    - Go to {% my network title="**Settings** > **System** > **Network**" %}, and under **Configure network interfaces**, make sure that IPv6 is enabled.
+  - Avoid using Matter in a complex network setup with vLANS or mDNS responders. These are more likely to cause network issues than if you use Matter in a flat network.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]: this one is also in the troubleshooting section - can we do something smart with just linking there. Something like "Make sure your home network meets the requirements" --> link to the troubleshooting or advanced details

File: source/_integrations/matter.markdown [hunk 2]
```diff
@@ -120,15 +127,14 @@ Each Matter network is called a fabric. Each home automation controller that con
 - Have either an Android or iPhone ready and Bluetooth enabled. For information why Bluetooth is required, refer to the section on [Bluetooth used during commissioning](#bluetooth-used-during-commissioning):
   - Android:
     - Have an Android phone (a full Android, not F-Droid).
-    - Have the Home Assistant Companion app installed.
+    - Have the latest version of Home Assistant Companion app installed.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
    - Have the latest version of the Home Assistant Companion app installed.
```

File: source/_integrations/matter.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_integrations/matter.markdown]: haha, right. that's why I didn't include it the first time round. Already forgot. 

Linking: I don't like the idea of linking to the troubleshooting section here, though:
- If we think the majority of people will need to look into their network settings before starting, this is a true prereq and should be mentioned here. 
- If not, then it is better not to add a link that sends them off-track.

File: source/_integrations/matter.markdown [hunk 3]
```diff
@@ -120,15 +122,14 @@ Each Matter network is called a fabric. Each home automation controller that con
...
     - Have an Android phone (a full Android, not F-Droid).
-    - Have the Home Assistant Companion app installed.
+    - Have the latest version of the Home Assistant Companion app installed.
     - Have Google Home app installed on the Android.
     - We are not going to add the new device to Google Home. The app is needed because Google included the Matter SDK there.
     - If you are using Thread: Make sure there is a Thread border router device (Nest Hub v2 or Nest Wi-Fi Pro) present in your home network.
   - iPhone
-    - Version 16 or higher
+    - Have an iPhone version 16 or higher
+    - Have the latest version of Home Assistant Companion app installed.
```
Comment (c0ffeeca7) [inline source/_integrations/matter.markdown]:
```suggestion
    - Have the latest version of the Home Assistant Companion app installed.
```

File: source/_integrations/matter.markdown [hunk 4]
```diff
@@ -120,15 +122,14 @@ Each Matter network is called a fabric. Each home automation controller that con
...
   - Android:
     - Have an Android phone (a full Android, not F-Droid).
-    - Have the Home Assistant Companion app installed.
+    - Have the latest version of the Home Assistant Companion app installed.
     - Have Google Home app installed on the Android.
     - We are not going to add the new device to Google Home. The app is needed because Google included the Matter SDK there.
     - If you are using Thread: Make sure there is a Thread border router device (Nest Hub v2 or Nest Wi-Fi Pro) present in your home network.
   - iPhone
-    - Version 16 or higher
+    - Have an iPhone version 16 or higher
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
    - Have the iOS version 16 or higher
```

File: source/_integrations/matter.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_integrations/matter.markdown]: removed.

File: source/_integrations/matter.markdown [hunk 5]
```diff
@@ -104,10 +104,12 @@ Each Matter network is called a fabric. Each home automation controller that con
 
 ### Prerequisites
 
+- Make sure you have the latest version of Home Assistant installed.
 - On the device packaging, check for both the Matter logo and for either the Wi-Fi or the Thread logo.
 - Check if the QR code is only on the packaging or if it is also on the device.
-  - If it is only on the packaging, snap a picture of the QR code and the device and store it in a save place.
+  - If it is only on the packaging, snap a picture of the QR code and the device and store the image and the numerical code in a save place.
   - If you lose the QR code and disconnect the device at some point, you won't be able to connect to that device again without the QR code.
+- If you are adding a Wi-Fi-based Matter device: Matter devices use the 2.4&nbsp;GHz frequency for Wi-Fi. For this reason, make sure your phone is in the same 2.4&nbsp;GHz network where you want to operate your devices.
```
Comment (agners) [inline source/_integrations/matter.markdown]:
```suggestion
- If you are adding a Wi-Fi-based Matter device: Matter devices often use the 2.4&nbsp;GHz frequency for Wi-Fi. For this reason, make sure your phone is in the same 2.4&nbsp;GHz network where you want to operate your devices.
```

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**